<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resetPassword</class>
 <widget class="QWidget" name="resetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اعادت تعيني كلمة المرور</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pathIcon.qrc">
    <normaloff>:/images/resetPassword.png</normaloff>:/images/resetPassword.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Arabic" country="SaudiArabia"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../pathIcon.qrc">:/images/resetPassword.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUsername">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>اسم المستخدم</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEmail">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>البريد الالكتروني</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textNumberID">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>الرقم الوطني</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>استعاده</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginReset">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>تسجيل الدخول</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textNewPasword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>كلمة المرور</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textCheckPassword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>اعادة كتابت كلمة المرور</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOkRestPassword">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>تعين كلمة المرور</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textaddCode">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ادخل الكود</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckCode">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>تحقق</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../pathIcon.qrc"/>
 </resources>
 <connections/>
</ui>
